modules/dnn_superres/tutorials/upscale_image_single/upscale_image_single.markdown

```diff
@@ -0,0 +1,66 @@
+Upscaling images: single-output {#tutorial_dnn_superres_upscale_image_single}
+===========================
+
+In this tutorial you will learn how to use the 'dnn_superres' interface to upscale an image via pre-trained neural networks.
+
+Building
+----
+
+When building OpenCV, run the following command to build the 'dnn_superres' module:
+
+```make
+cmake -DOPENCV_EXTRA_MODULES_PATH=<opencv_contrib>/modules -Dopencv_dnn_superres=ON <opencv_source_dir>
```

Comment:
> -Dopencv_dnn_superres=ON

There is no such option.
There are similar `BUILD_` options, but adding them here is redundant (`=ON` be default).

It is better to mention `-DBUILD_EXAMPLES=ON` here and in other tutorials (`=OFF` be default).